scripts/install

```diff
@@ -128,28 +67,28 @@ ynh_install_composer
 #=================================================
 ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-app_url_domain="https://$domain${path_url%/}"
+app_url_domain="https://$domain${path%/}"
 
-ynh_add_config --template=../conf/.env.example --destination=$final_path/.env
+ynh_add_config --template=../conf/.env.example --destination=$install_dir/.env
 
 #=================================================
 # FINAL BOOKSTACK INSTALL
 #=================================================
 ynh_script_progression --message="Install BookStack" --weight=5
 
-pushd $final_path
+pushd $install_dir
 	php$phpversion artisan key:generate --no-interaction --force
 	php$phpversion artisan migrate --no-interaction --force
 	php$phpversion artisan bookstack:create-admin --email="$email" --name="$admin" --external-auth-id="$admin"
```

Comment:
I just tried installing it to debug the multi-instance install and the POST_INSTALL message is saying

>For the first time Login with standard authentication, use the default credentials `admin@admin.com` and `password`. You should change these details immediately after logging in for the first time.

But this seem to contradict what is done here where an admin user is created using the `$admin` var ? o_O

Reply:
Ah my bad this is about "standard auth", like if somebody disables LDAP auth